webnode/src/utils/encryption.js

```diff
@@ -86,8 +89,29 @@ const forgeEncrypt = (key, secret, nonce) => {
   return encrypted.toHex() + tag.toHex();
 };
 
```

Comment:
decryptTreasure is just a wrapper so you don't have to do any trimming or converting of anything before attempting to decrypt.  Can give it the signatureMessageFragment as-is.